lib/src/core/authentication-core.ts

```diff
@@ -286,32 +293,34 @@ export class AuthenticationCore<T> {
                     "revokeAccessToken",
                     "No revoke access token endpoint found.",
                     "No revoke access token endpoint was found in the OIDC provider meta data returned by " +
-                        "the well-known endpoint or the revoke access token endpoint passed to the SDK is empty."
+                    "the well-known endpoint or the revoke access token endpoint passed to the SDK is empty."
                 )
             );
         }
 
         const body: string[] = [];
-        body.push(`client_id=${configData.clientID}`);
-        body.push(`token=${(await this._dataLayer.getSessionData()).access_token}`);
+        body.push(`client_id=${ configData.clientID }`);
+        body.push(`token=${ (await this._dataLayer.getSessionData()).access_token }`);
         body.push("token_type_hint=access_token");
 
-        return axios
-            .post(revokeTokenEndpoint, body.join("&"), {
-                headers: AuthenticationUtils.getTokenRequestHeaders(),
-                withCredentials: configData.sendCookiesInRequests
-            })
+        return fetch(revokeTokenEndpoint, {
+            body: body.join("&"),
+            // eslint-disable-next-line max-len
+            credentials: configData.sendCookiesInRequests ? FetchCredentialTypes.Include : FetchCredentialTypes.SameOrigin,
+            headers: new Headers(AuthenticationUtils.getTokenRequestHeaders()),
+            method: "POST"
+        })
             .then((response) => {
-                if (response.status !== 200) {
+                if (!response.ok) {
```

Comment:
Is there any reason for using `ok` instead of the status?

Reply:
In the documentations, they have used `response.ok` to check the network status but Its better to use  `response.status !== 200`